libs/ml/include/ml/distributed_learning/distributed_learning_client.hpp

```diff
@@ -349,9 +349,9 @@ void TrainingClient<TensorType>::AddGradient(GradientType &gradient)
 template <class TensorType>
 void TrainingClient<TensorType>::AddExportGradient(GradientType &gradient)
 {
-  std::unique_lock<std::mutex> l(export_buffer_cv_mutex_);
+  std::unique_lock<std::mutex> lock(export_buffer_cv_mutex_);
```

Comment:
??

Reply:
Purely for readability. Lowercase L == ++ungood

Reply:
Nope sorry I should have made a better comment - I am curious about this piece of code because there is a mutex for the condition along with another mutex. Maybe a question for @baykaner which will be ultimately update in the future